teams/wg-const-eval.toml

```diff
@@ -0,0 +1,19 @@
+name = "wg-const-eval"
+subteam-of = "compiler"
+wg = true
+
+[people]
+leads = ["oli-obk", "RalfJung"]
+members = [
+    "ecstatic-morse",
+    "oli-obk",
+    "RalfJung",
+]
+
+[website]
+name = "Compile-time function evaluation"
+description = "Expanding the capabilities of compile-time function evaluation in Rust"
```

Comment:
"and ensuring their soundness" or so? I do not entirely see my work reflected in this description. ;)

Reply:
Would you settle for an adverb? "Soundly expanding"?

Reply:
That sounds good, too. :D